eisbuk-admin/src/store/selectors/customers.ts

```diff
@@ -22,3 +23,46 @@ export const getCustomersList = (state: LocalStore): Customer[] => {
   const customersInStore = getCustomersRecord(state);
   return Object.values(customersInStore);
 };
+
+/**
+ * Creates a selector that gets a list of all the customers whose birthdays are today
+ * (not an actual firestore query but rather synced local store entry)
+ * @param date DateTime to use as "today"
+ * @returns selector to get a list of customers grouped by birthday
+ */
+export const getCustomersWithBirthday =
+  (date: DateTime) =>
+  (state: LocalStore): CustomersByBirthday[] => {
+    const customersInStore = getCustomersRecord(state);
+
+    const customers: CustomersByBirthday[] = [];
+    Object.values(customersInStore).forEach((customer) => {
+      const index = customers.findIndex(
+        (customerBirthday) => customerBirthday.birthday === customer.birthday
+      );
+      index !== -1
+        ? customers[index].customers.push(customer)
+        : customers.push({
+            birthday: customer.birthday,
+            customers: [customer],
+          });
+    });
+    const sortedCustomers = customers.sort((a, b) =>
+      DateTime.fromISO(a.birthday)
+        .set({ year: 2000 })
+        .toString()
+        .localeCompare(
+          DateTime.fromISO(b.birthday).set({ year: 2000 }).toString()
+        )
+    );
+    const index = sortedCustomers.findIndex(
+      (c) =>
+        date.startOf("day").toString().substring(5) <=
+        DateTime.fromISO(c.birthday).startOf("day").toString().substring(5)
```

Comment:
Right, so...no `startOf` setting... you get a birthday string, for instance `"2021-08-01"` and a different ISO string to compare, for instance "2020-05-22". Then you just compare the two: 
```typescript
const compareDateOfYear = (a: string, b: string) => a.substr(5) <= b.substr(5) 
```
You can use a version of this for each date comparison within the selector, so I suggest you either convert the `date: DateTime` input to ISO string at the beginning of the selector and just use the `dateISO` or require the `date` param to be `date: string` (and do the conversion outside the selector)